### DISCUSSION TITLE
No puedo ingresar al grupo de microsoft teams

### DISCUSSION THREAD

**Discussion by qwertyGuyn** [Launch-X-Latam/MisionFrontEnd#37] [Q&A]
Hola, estuve intentando ingresar al grupo durante el onboarding pero cuando respondí el formulario, busqué en el correo que dieron y ahí estaba la invitación, cuando le doy clic, en teams me sale que "solicitud enviada" y que solo hay un miembro. ¿Qué puedo hacer?

**Comment by kilerite** [comment]
hola estoy igual, no puedo acceder porque me dice que tengo un problema con la contraseña y necesito recordarla pero no me deja.. me gustaria que me puedan ayudar. gracias !

**Comment by Pablo-Jaramillo** [comment]
Envíen un correo a innovaccion, aquí se los dejo (si responden, así que no se preocupen) contacto@innovaccion.mx
También pueden mandar un mensaje a su cuenta de Instagram, pero escriban lo más detallado y con imágenes si es posible para que sea más ágil la resolución.

**Reply by kilerite** [reply]
ya lo hice por los 2 medios pero no me responden.. y no puedo entrar al teams.  simplemente la cuenta microsoft no puedo acceder ..

**Reply by Pablo-Jaramillo** [reply]
Bueno, al correo mandan mensaje para entrar al grupo en el que anexan este link https://bit.ly/3BWOxdE después de darle clic te pide ingresar, pones la cuenta y la contraseña a la que cambiaste cuando accediste por primera vez. Si olvidaste la contraseña, puedes intentar restablecerla aunque no se si te lo permita. Si eso ultimo es el problema, que no te deja acceder por olvidar la contraseña entonces solo habrá que esperar a que te respondan.

Si nunca habías accedido a la cuenta, recuerda que en tu correo de acceso te dan la información de la cuenta y la contraseña para acceder por primera vez, pero después te pide cambiar la contraseña.
